docs/notification-routing/routing-by-time.rst

```diff
@@ -0,0 +1,643 @@
+Time-limiting Sink Activity
+=============================================
+
+It is possible, for any sink, to set the schedule of its activation by specifying the ``activity`` field in its
+configuration. You can specify multiple time spans, with specific days of the week and hours in these days that
+the sink will be active. Outside of these specified time spans, the sink will not run - so for example Slack
```

Comment:
mentioning Slack here as an example is confusing because it lacks context. Either remove the offhand reference to slack or introduce it properly ("for example, if you use Slack for notifications and the Slack sink has an activity block specified than you can avoid sending slack messages at certain hours")